suites/nautilus/upgrades/tier-2_upgrade_with-tags.yaml

```diff
@@ -176,47 +192,53 @@ tests:
               mon_allow_pool_delete: true
             client:
               rgw crypt require ssl: false
-              rgw crypt s3 kms encryption keys: testkey-1=YmluCmJvb3N0CmJvb3N0LWJ1aWxkCmNlcGguY29uZgo=
+              rgw crypt s3 kms encryption keys:
+                testkey-1=YmluCmJvb3N0CmJvb3N0LWJ1aWxkCmNlcGguY29uZgo=
                 testkey-2=aWIKTWFrZWZpbGUKbWFuCm91dApzcmMKVGVzdGluZwo=
-    desc: Rolling update 4.x with tags mons
-    abort-on-fail: True
+      desc: Rolling update 4.x with tags mons
+      abort-on-fail: True
 
-- test:
-    desc: "generate sosreport"
-    module: sosreport.py
-    name: "generate sosreport"
+  - test:
```

Comment:
can you please re-consider about the need of sos report generation after every upgrade test case?

Reply:
this is a customer scenario where sos report is required at each step to get ceph mon osd dumps etc